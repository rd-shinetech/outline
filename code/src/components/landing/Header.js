import React from 'react';



const Header = () => {
    return (
        <header>
            <div className="jumbotron jumbotron-fluid bg-info text-white text-center">
                <div className="container">
                    <h1 className="display-4">ReactJS CRUD Boilerplate</h1>
                    <p className="lead">with mock backend</p>
                </div>
            </div>
        </header>
    );
};


export default Header;
