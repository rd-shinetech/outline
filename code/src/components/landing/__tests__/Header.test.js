import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../Header';


describe('Header.test.js', () => {

    it('renders without crashing', () => {
        
        const wrapper = shallow(<Header />);

        expect(wrapper).toHaveLength(1);

        expect(wrapper.find('header')).toHaveLength(1);

        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();        
        
    });
});
