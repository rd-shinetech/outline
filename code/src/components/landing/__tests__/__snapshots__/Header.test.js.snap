exports[`Header.test.js renders without crashing 1`] = `
<header>
  <div
    className="jumbotron jumbotron-fluid bg-info text-white text-center">
    <div
      className="container">
      <h1
        className="display-4">
        ReactJS CRUD Boilerplate
      </h1>
      <p
        className="lead">
        with mock backend
      </p>
    </div>
  </div>
</header>
`;
